portal/src/test/scala/cromwell/pipeline/datastorage/dao/repository/UserRepositoryTest.scala

```diff
@@ -50,6 +51,16 @@ class UserRepositoryTest extends AsyncWordSpec with Matchers with BeforeAndAfter
       }
     }
 
-  }
+    "getUsersByEmail" should {
 
+      "should find newly added user by email pattern" taggedAs (Dao) in {
+        val newUser: User = TestUserUtils.getDummyUser()
+        userRepository
+          .addUser(newUser)
+          .flatMap(
+            _ => userRepository.getUsersByEmail(newUser.email).map(repoResp => repoResp shouldBe Vector(newUser))
```

Comment:
Checking that the method returns a `Vector` is not a good idea since this is not guaranteed and may cause tests to fail if the method will return, for example, a `List`. It is better to use things like `should contain theSameElementsAs`.

Reply:
As I got it, I can use - `repoResp should contain theSameElementsAs Vector(newUser)` - instead, Isn't it?